README.md

```diff
@@ -305,6 +305,56 @@ in your system configuration and
 
 in your Home Manager configuration.
 
+Flakes
+------
+
+Home Manager includes a flake.nix for compatibility with [NixOS flakes](https://nixos.wiki/wiki/Flakes) that wish
+to use it as a module. A bare-minimum flake.nix would be as follows:
+
+```nix
+{
+  description = "NixOS configuration";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    home-manager.url = "github:rycee/home-manager";
+  };
+
+  outputs = inputs: {
+    nixosConfigurations = {
+      hostname = let
+        system = "x86_64-linux";
+        pkgs = inputs.nixpkgs.legacyPackages.${system};
+        inherit (inputs.nixpkgs) lib;
+
+        specialArgs = {
+          inherit inputs;
+        }; # Things in this set are passed to modules and accessible in the top-level arguments (e.g. `{ pkgs, lib, inputs, ... }:`)
+
+        hm-nixos-as-super = { config, ... }: {
+          # Submodules have merge semantics, making it possible to amend the `home-manager.users` submodule for additional functionality
```

Comment:
Might help readability to break these long comments?

Reply:
Fair point! Should look better now :)